docs/monitoring/metrics.md

```diff
@@ -1470,19 +1470,11 @@ Certain RocksDB native metrics are available but disabled by default, you can fi
       <td>Gauge</td>
     </tr>
     <tr>
-      <th rowspan="4"><strong>Operator</strong></th>
-      <td>currentInput1Watermark</td>
```

Comment:
But we are not deprecating or removing it. There used to be just `currentInput1Watermark` and `currentInput2Watermark`. Since then I've added `currentInput3Watermark`, ...,  `currentInputNWatermark`

Reply:
Right, I somehow missed it :man_shrugging: